src/lib.rs

```diff
@@ -6,34 +6,45 @@ mod encoding;
 mod encryption_layer;
 mod error;
 mod findex;
+mod memory;
 mod ovec;
 mod secret;
 mod symmetric_key;
+#[cfg(any(test, feature = "test-utils"))]
+mod test;
 mod value;
 
 pub use address::Address;
 pub use adt::{IndexADT, MemoryADT};
+#[cfg(feature = "bench")]
+pub use encoding::{Op, WORD_LENGTH};
+#[cfg(feature = "test-utils")]
+pub use encoding::{dummy_decode, dummy_encode};
+pub use error::Error;
 pub use findex::Findex;
+#[cfg(feature = "redis-store")]
+pub use memory::db_stores::redis_store::RedisStore;
```

Comment:
I would like not to see "store" anymore. Remove this `db_store` and share your error type with `in_memory.rs`.